pom.xml

```diff
@@ -28,18 +28,34 @@
         <selenium.version>3.1.0</selenium.version>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.amazonaws</groupId>
+                <artifactId>aws-java-sdk-bom</artifactId>
```

Comment:
Looking at the POM for this, it seems to include literally all the AWS APIs:

https://search.maven.org/#artifactdetails%7Ccom.amazonaws%7Caws-java-sdk-bom%7C1.11.155%7Cpom

I see a dependency on "aws-java-sdk-s3" further down - is "aws-java-sdk-bom" still used?